templates/careers/apply.html

```diff
@@ -0,0 +1,139 @@
+{% extends '/careers/careers_base.html' %}
+
+{% block hero %}
+  <section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom u-extra-space">
+    <div class="p-strip--suru-background">
+      <div class="row">
+        <div class="col-6">
+          <h1>Apply</h1>
+        </div>
+      </div>
+    </div>
+  </section>
+  <style>
+    .p-strip--suru-background {
+      background-image:
+      linear-gradient(to bottom right, rgba(74, 24, 60, 0.4) 0%, rgba(74, 24, 60, 0.4) 49.9%, rgba(74, 24, 60, 0) 50%, rgba(74, 24, 60, 0) 100%),
+      linear-gradient(to top right, rgba(210, 180, 202, 0) 0%, rgba(210, 180, 202, 0) 49.9%, rgba(210, 180, 202, 0.05) 50%, rgba(210, 180, 202, 0.05) 100%),
+      linear-gradient(to top right, #fff 0%, #fff 49%, transparent 50%),
+      linear-gradient(140deg, #E95420 0%, #772953 33%, #2C001E 72%);
+      background-position: left top, right top, left 101%;
+      background-size: 80% 100%, 70% 80%, 110% 20%, 100% 100%;
+      padding-bottom: 6rem;
+      padding-top: 6rem;
+    }
+  </style>
+{% endblock %}
+
+{% block overview %}
+  <div class="row u-sv3">
+    <div class="col-12">
+      <a href="{{ request.referrer }}">&lsaquo;&nbsp;Back to list</a>
+    </div>
+  </div>
+  <div class="row u-extra-space">
+    <div class="col-12">
+      <div class="p-card--strip" style="margin-bottom: 0;">
+        <h3>Roles applying for:</h3>
+        <div class="row u-sv3">
+          <div class="col-6">
+            <input type="checkbox" id="CheckMe1">
```

Comment:
These should have relative id's

Reply:
You are correct. I will add the relative id's when I will add the functionality to select jobs and then apply, in a separate PR.